pkg/virt-launcher/virtwrap/live-migration-source.go

```diff
@@ -918,6 +918,8 @@ func (l *LibvirtDomainManager) asyncMigrate(vmi *v1.VirtualMachineInstance, opti
 		if err != nil {
 			log.Log.Object(vmi).Reason(err).Error("Live migration failed.")
 			migrationErrorChan <- err
+
+			l.hotPlugHostDevices(vmi)
```

Comment:
I understand that we can not do much if this fails, but it may be useful to log it as an error if it occurs.

Reply:
Done